sensor/common/compliance/nodescan_handler.go

```diff
@@ -4,21 +4,24 @@ import (
 	"github.com/stackrox/rox/generated/internalapi/central"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/sync"
 	"github.com/stackrox/rox/sensor/common"
 )
 
 // NodeScanHandler is responsible for handling the arriving NodeScanV2 messages, processing then, and sending them to central
 type NodeScanHandler interface {
 	common.SensorComponent
+	Stopped() concurrency.ReadOnlyErrorSignal
 }
 
 // NewNodeScanHandler returns a new instance of a NodeScanHandler
 func NewNodeScanHandler(ch <-chan *storage.NodeScanV2) NodeScanHandler {
 	return &nodeScanHandlerImpl{
-		nodeScans: ch,
-		toCentral: make(chan *central.MsgFromSensor),
-
-		stopC:    concurrency.NewErrorSignal(),
-		stoppedC: concurrency.NewErrorSignal(),
+		nodeScans:    ch,
+		toCentral:    make(chan *central.MsgFromSensor),
+		stopC:        concurrency.NewErrorSignal(),
```

Comment:
This will not be consistent with how `stopC`/`stoppedC` are named in other files. Either we should keep the old wrong names here or we should rename all occurrences proper way. The worse thing is to have multiple patterns that mean the same thing.